### ISSUE TITLE
send-keys enter not working as expected

### ISSUE THREAD

**Issue by nkh** [tmux/tmux#1778]
enter doesn't seem to send enter to the pane. 

the code, executed from routine "create" in a shell script:
    tmux send-keys -t "$mirror:0" "$command" enter

executed as: "$ tmuxslider create iotop" 
   where tmuxslider is the bash script and 'iotop' is put in $command and should be executed

the pane contains this after execution: 
iotop
$ iotop

the command is displayed, then the prompt is displayed by bash, then the command passed by send-keys but enter is not send, ie iotop is not executed

I changed the code to:     tmux send-keys -t "$mirror:0" "$command" enter enter enter
   two enter  added at the end of  the call

executed as: "$ tmuxslider create iotop" 
the pane contains this:
iotop

$ iotop
.git/            ideas            README.md        .tmuxslider.swp
.gitignore       LICENSE          tmuxslider       Todo.txt
$ iotop .gitignore

the command is displayed
2 blank lines are displayed
a prompt with the command is displayed
file completion for the directory is displayed (!)
a prompt is displayed with the command and the first file that can be completed (!)

no return, ie the command is not ecexuted

Although I doubt this is a tmux only problem (too much weird stuff displayed), I fail to see any reason or solution, hints appreciated.

I have simplified  my .bashrc so the execution is as fast as possible, same results.
 
#information
tmux 2.7
Linux x86_64
xterm-256color

**Comment by nicm** [comment]
Is this actually all the script does? Does it also create the pane in the
first place? Because I can't reproduce it just with the command you have
given.

Probably the enter is being sent before the shell has started and it is
eating it. Try adding a sleep between the new-window/split-window and the
send-keys.

**Comment by nkh** [comment]
It did work with a delay of .4 s. thanks.

weird trying to make my shell run faster I have emptied my .bashrc, a rather large one, and used a simplified prompt and it failed. I have to dig more into this, I have noticed that another of my tmux scripts doesn't work with send-keys either, must be the same problem.

script is here: https://github.com/nkh/tmuxslider
$ tmuxslider --help | more  

there's a quick-start command to try without installing anything
